trax/data/tf_inputs_test.py

```diff
@@ -591,21 +591,50 @@ def test_create_bert_inputs(self):
     create_inputs_2 = tf_inputs.CreateBertInputs(True)
     for res in create_inputs_1(zip(inputs_sentences_1, labels)):
       values, segment_embs, _, label, weight = res
-      self.assertTrue((values == np.array([101, 100, 150, 200, 102])).all())
-      self.assertTrue((segment_embs == np.zeros(5)).all())
+      self.assertAllEqual(values, np.array([101, 100, 150, 200, 102]))
+      self.assertAllEqual(segment_embs, np.zeros(5))
       self.assertEqual(label, np.int64(1))
       self.assertEqual(weight, np.int64(1))
 
     for res in create_inputs_2(
         zip(inputs_sentences_1, inputs_sentences_2, labels)):
       values, segment_embs, _, label, weight = res
-      self.assertTrue(
-          (values == np.array([101, 100, 150, 200, 102, 300, 500, 102])).all())
+      self.assertAllEqual(values, np.array([101, 100, 150, 200, 102, 300, 500, 102]))
       exp_segment = np.concatenate((np.zeros(5), np.ones(3)))
-      self.assertTrue((segment_embs == exp_segment).all())
+      self.assertAllEqual(segment_embs, exp_segment)
       self.assertEqual(label, np.int64(1))
       self.assertEqual(weight, np.int64(1))
 
+  def test_mask_random_tokens(self):
```

Comment:
a few things here:

- it again seems like 103 is MASK?
- why is 101 special (and why it isn't being masked), let's give it its name "CLS"? what?
- we are checking that sum(out == 103) <= 15 ... but also make sure that it is > 0?

Reply:
i gave all magic numbers names. CLS isn't masked because in real scenario it marks the beginning of the sentence and is used to get sentence embedding. Here there are a lot of CLS to test whether function won't mask CLS and whether the number of masked tokens won't be infected by that.

sum(out == MASK_id) can be equal to zero. For each candidated it is randomly chosen whether it is masked (80% probability) changed to random token (10%) and unchanged (10%). I didn't set the seed here so it might (although unlikely) happen that none of the tokens is are masked.